about.html

```diff
@@ -4,9 +4,107 @@
+           <div class="mobile-menu-separator"></div>
+           <li><a href="#sponsor">Sponsor</a></li>
+           <div class="mobile-menu-separator"></div>
+        </ul>
+      </div>
+    </div>
+  </header>
+      <section class="about-headline">
+      <div class="introduction introduction-about">
+         <h1 class="heading-title"> Web development course For 2023</h1>
+         <p class="description about-description">Online Course’s mission is to provide flexible, effective skill development to empower organizations and individuals. <br>
+          Online Course’ Business helps companies achieve critical business outcomes and stay competitiv.e by offering engaging on-demand, immersive, and cohort-based learning.
+        </p>
+         <p class="contact"> Please Contact us per Email for any further questions about Online Course! <br> <a href="">online@course.org</a>  
+         </p>
+      </div>
+    </section>
+    <section id="about-logo">
+      <p class="logo-title">Online Course 2023 logo</p>
+      <div class="logo-separator"></div>
+      <p class="logo-description">The Logo of Online Course 2023 was decided through the logo competition from 8. June to 7. July. <br> After online public voting and Selection Committee's judging process.the log from Nagesh Agrawal wond the price </p>
+      <div class="logo-image">
+     <img src="./assets/images/online-learning-course-home-schooling-education-and-graduation-logo-free-vector.webp" alt="online course logo" >
+      </div>
+    </section>
+    <section id="past-courses">
+      <h2 class="past-title"> See our past courses</h2>
+      <div class="past-separator"></div>
+      <p class="past-description">Take a look at the courses of the last two years</p>
+      <ul class="course-gallery">
+        <li class="year">
+          <div>
+          <h3>2022</h3>
+          <p>2022 web development course</p>
+        </div>
+        </li>
+        <li class="year">
+          <div>
+          <h3>2021</h3>
+          <p>2021 web development course</p>
+        </div>
+        </li>
+      </ul>
+    </section>
+  <section id="sponsor">
+    <h3 class="program-title">Partner</h3>
+    <div class="program-separator"></div>
+    <div class="sponsor-list">
+     <div class="sponsor-row">
+      <a href="https://www.mozilla.org/en-US/foundation/">mozilla</a>
+      <a href="https://www.google.com/">Google</a>
+      <a href="http://www.naver.com/">Naver</a>
+     </div>
+       <div class="sponsor-row">
+        <a href="http://www.kakaocorp.com/">daumKakao</a>
+        <a href="https://www.airbnb.co.kr/">airbnb</a>
+       </div>
+    </div>
+  </section>
+  <footer>
+    <a id="logo" href=""><img src="./assets/images/online-learning-course-home-schooling-education-and-graduation-logo-free-vector.webp" alt="online course logo"  width="100"></a>
+    <span class="footer-description">2023 Online Course Web development. Some Rights Reserved.</span>
+  </footer>
```

Comment:
- [ ] The footer of the About Page has a dark background, please kindly change the color of the footer of the about page🙃. But the background color of the footer for the mobile version is white and not black 😉 

Your design

<img width="964" alt="Screen Shot 2023-02-07 at 23 01 43" src="https://user-images.githubusercontent.com/74501523/217352757-b6d2f1d4-892c-436c-a6e1-7f06457c5a01.png">

Cindy's design

<img width="874" alt="Screen Shot 2023-02-07 at 23 01 17" src="https://user-images.githubusercontent.com/74501523/217352893-cafdb657-9d81-4730-a88e-81716c8acfca.png">